src/routes/tools/extreme-precipitation/_store.js

```diff
@@ -0,0 +1,101 @@
+import { writable, derived } from "svelte/store";
+import climvars from "~/helpers/climate-variables";
+import { makeCustomWritableStore } from "../_common/stores";
+import {
+  CLIMATE_INDICATORS,
+  DEFAULT_CLIMATE_VARIABLE,
+  DEFAULT_CLIMATE_INDICATOR,
+  DEFAULT_DURATION,
+  DEFAULT_RETURN_PERIOD,
+  DEFAULT_THRESHOLD_TYPE,
+} from "./_constants";
+
+import {
+  calcHeatwaveCount,
+  calcMaxDuration,
+  calcDaysCount,
+  groupDataByYear,
+} from "./_data";
+
+export const climvarStore = makeCustomWritableStore(DEFAULT_CLIMATE_VARIABLE, {
+  name: "climvarStore",
+  getters: [
+    {
+      name: "climvar",
+      getter: ($s) => climvars.find((d) => d.id === $s),
+    },
+  ],
+});
+
+export const indicatorList = CLIMATE_INDICATORS;
```

Comment:
Would it make sense to make `indicatorList` a readable store if it's never updated?

Reply:
I've left `indicatorList` a constant, to match the other lists used to populate settings e.g. `climvars`, `scenarios`, etc.